services/user_interface/templates/rightornot.html

```diff
@@ -53,17 +48,12 @@
     function msg(direction) {
       // finished labelling round
       if(swipeCount >= maxSwipes) {
+        $("#results_box").html("")
         $("#political_samples_box").html(
-              `
-              <p>Wir zeigen einige politische Texte. </p>
-              <p>
-              Swipe
-              <ul>
-                  <li><b>links</b> wenn Du den Text links fandest</li>
-                  <li><b>rechts</b> wenn Du den Text rechts fandest</li>
-                  <li><b>runter</b> wenn Du den Text neutral fandest</li>
-              </ul>`
-        );
+            `<p>Du hast ` + (100 * wrong / (maxSwipes / 2)).toPrecision(3) + "% falsch eingeordnet.</p>" +
```

Comment:
ich wuerd nicht sagen falsch sondern "anders als im datensatz" oder so

Reply:
stimmt, is besser. 

hatte auch ueberlegt, sowas hier fuer den bias zu plotten:
https://codepen.io/plotly/pen/rxeZME
aber fand ich dann doch zu plakativ irgendwie.

Reply:
ja vielleicht schon, ein einfacher bar chart anstelle?